src/eei.cpp

```diff
@@ -492,18 +492,23 @@ namespace hera {
       evmc_result call_result;
 
       // Start with base call gas
-      int64_t extra_gas = GasSchedule::call;
+      takeInterfaceGas(GasSchedule::call);
 
       // Charge valuetransfer gas if value is being transferred.
       // Only charge callNewAccount gas if the account is new and value is being transferred per EIP161.
       if (!isZeroUint128(call_message.value)) {
-        extra_gas += GasSchedule::valuetransfer;
+        takeInterfaceGas(GasSchedule::valuetransfer);
         if ((kind == EEICallKind::Call) && !m_context->fn_table->account_exists(m_context, &call_message.destination))
-          extra_gas += GasSchedule::callNewAccount;
+          takeInterfaceGas(GasSchedule::callNewAccount);
       }
 
-      // This check is in EIP150 but not in the YellowPaper
```

Comment:
This comment `// This check is in EIP150 but not in the YellowPaper` appears to have disappeared, but I find it helpful. Could you add it back above the two previous `takeInterfaceGas` calls?